test/test_pebble.py

```diff
@@ -1874,189 +1885,194 @@ def test_change_error(self):
         })
         change = build_mock_change_dict()
         change['err'] = 'Some kind of service error'
-        self.client.responses.append({
+        client.responses.append({
             "result": change,
             "status": "OK",
             "status-code": 200,
             "type": "sync"
         })
         with pytest.raises(pebble.ChangeError) as excinfo:
-            self.client.autostart_services()
+            client.autostart_services()
         assert isinstance(excinfo.value, pebble.Error)
         assert excinfo.value.err == 'Some kind of service error'
         assert isinstance(excinfo.value.change, pebble.Change)
         assert excinfo.value.change.id == '70'
 
-        assert self.client.requests == [
+        assert client.requests == [
             ('POST', '/v1/services', None, {'action': 'autostart', 'services': []}),
             ('GET', '/v1/changes/70/wait', {'timeout': '4.000s'}, None),
         ]
 
-    def test_wait_change_success(self, timeout: typing.Optional[float] = 30.0):
+    def test_wait_change_success(self, client: MockClient, timeout: typing.Optional[float] = 30.0):
         change = build_mock_change_dict()
-        self.client.responses.append({
+        client.responses.append({
             "result": change,
             "status": "OK",
             "status-code": 200,
             "type": "sync"
         })
 
-        response = self.client.wait_change(pebble.ChangeID('70'), timeout=timeout)
+        response = client.wait_change(pebble.ChangeID('70'), timeout=timeout)
         assert response.id == '70'
         assert response.ready
 
-        assert self.client.requests == [
+        assert client.requests == [
             ('GET', '/v1/changes/70/wait', {'timeout': '4.000s'}, None),
         ]
 
-    def test_wait_change_success_timeout_none(self):
-        self.test_wait_change_success(timeout=None)
+    def test_wait_change_success_timeout_none(self, client: MockClient):
+        self.test_wait_change_success(client, timeout=None)
 
-    def test_wait_change_success_multiple_calls(self):
+    def test_wait_change_success_multiple_calls(self, client: MockClient, time: MockTime):
```

Comment:
I'm still of the opinion that we could probably combine the client and time fixtures into a single one (exposing some time functionality as required). If you disagree, this would probably be a good quick discussion tomorrow morning :)

If we keep them separate, I would still definitely rather see `time` renamed - if I see `time` I really expect it to either be the `time` module or the `time.time()` function.

Reply:
After discussion, we decide to kee the `time` fixture.